### ISSUE TITLE
Uncorrect architecture GraphConvModel

### ISSUE THREAD

**Issue by Storm35** [deepchem/deepchem#2670]
## Bug

I'm trying to reproduce my old deepchem GraphConvModels (from tensorgraph...) with the Keras models from the new version.
I used the same hyperparameters as the old models but metrics are far worst than the older ones.
I tried to look at the architecture and it looks like it's not well defined.

`model.model.summary() `  gives the following architecture:  
```
Model: "private__graph_conv_keras_model"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
graph_conv (GraphConv)       multiple                  204288    
_________________________________________________________________
graph_conv_1 (GraphConv)     multiple                  346752    
_________________________________________________________________
graph_conv_2 (GraphConv)     multiple                  346752    
_________________________________________________________________
batch_normalization (BatchNo multiple                  512       
_________________________________________________________________
batch_normalization_1 (Batch multiple                  512       
_________________________________________________________________
batch_normalization_2 (Batch multiple                  512       
_________________________________________________________________
batch_normalization_3 (Batch multiple                  1024      
_________________________________________________________________
graph_pool (GraphPool)       multiple                  0         
_________________________________________________________________
graph_pool_1 (GraphPool)     multiple                  0         
_________________________________________________________________
graph_pool_2 (GraphPool)     multiple                  0         
_________________________________________________________________
dense (Dense)                multiple                  33024     
_________________________________________________________________
graph_gather (GraphGather)   multiple                  0         
_________________________________________________________________
trim_graph_output (TrimGraph multiple                  0         
_________________________________________________________________
dense_1 (Dense)              multiple                  513       
=================================================================
Total params: 933,889
Trainable params: 932,609
Non-trainable params: 1,280
```

I expected the convolution layers to use a pattern like :
 - GraphConv
 - BatchNormalisation
 - (Dropout) here=0
 - GraphPool
Then
 - GraphConv again
 - ...

In the _GraphConvKerasModel used to define the model, the `call()` method looks well implemented to build the required architecture but doesn't seem to be used anywhere.

Difference between metrics:
Old model: r2score=0.83 MAE=31.19
New_model: r2score=0.75  MAE=40.34

Do I need to define my own KerasModel with requested architecture ?

## To Reproduce

 ```python
model = GraphConvModel(1, dense_layer_size=256, graph_conv_layers=[128, 128, 128], learning_rate=0.0005, batch_size=64, mode='regression')
model.fit(trainset, nb_epochs=30)
model.model.summary()
```

## Environment

* OS: Ubuntu 18.04.5
* Python version: 3.7
* DeepChem version: 2.5.0
* TensorFlow version: 2.5.1

Old models:
* DeepChem version: 2.1.1
* Tensorflow version: 1.13.2

**Comment by Storm35** [comment]
This link explains why the summary is not good. But I still don't understand why the metrics are far worst.
https://github.com/tensorflow/tensorflow/issues/44591
